### PR TITLE
Add styling FavoritePage

### LINKED ISSUE #30: Add styling - Favorites
## Value proposition

**As a** user,
**I want to** have a well styled app
**so that** i can enjoy using the app

## Acceptance criteria

- [x] styling should be consistent all over the map
- [x] same elements/components should look the same all over the app (Buttons, Cards shown on different pages, bodyfont, icons)
- [x] font(s) should be easy to read
- [x] font size should not be specified in "px"

## Complexity

small to medium

## Tasks

- [x] create a branch
- [x] add styling
- [x] check for a11y

### PR DESCRIPTION
**Pull request by bensle**

### CODE REVIEW

File: src/components/Favorite/Favorite.js [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
+import {TbBuildingMonument} from 'react-icons/tb'; //culture
+import {TbBeach} from 'react-icons/tb'; //recovery
+import {TiDeleteOutline} from 'react-icons/ti';
 
-import {SpanAdd, FavButtonDelete} from '../ActivityCard/ActivityCardStyle';
 import Backdrop from '../ComfirmationModal/Backdrop';
 import ConfirmationModal from '../ComfirmationModal/ConfirmationModal';
 
-import {FavDiv, FavButton, Section} from './FavoriteStyle';
+import * as Style from './FavoriteStyle';
```
Comment (JessicaLoers) [inline src/components/Favorite/Favorite.js:15]: Nice one 😉

File: src/components/Favorite/Favorite.js [hunk 2]
```diff
@@ -21,27 +28,136 @@ export default function Favorite({
...
+                </Style.TypeSpan>
+              )}
+              {type === 'recovery' && (
+                <Style.TypeSpan>
+                  <TbBeach />
+                </Style.TypeSpan>
+              )}
+              {type === 'challenge' && (
+                <Style.TypeSpan>
+                  {' '}
```
Comment (JessicaLoers) [inline src/components/Favorite/Favorite.js]: I guess this  line breaks `{' '}` is caused by accidents, if so please remove. 
If not, then ignore this comment 😁

File: src/components/Favorite/Favorite.js [hunk 3]
```diff
@@ -21,27 +28,136 @@ export default function Favorite({
...
+                </Style.TypeSpan>
+              )}
+            </Style.MiddleLi>
+          )}
+          {duration === 'vacation' && (
+            <Style.LongLi>
+              <MdOutlineAccessTimeFilled />
+              {type === 'culture' && (
+                <Style.TypeSpan>
+                  {' '}
```
Comment (JessicaLoers) [inline src/components/Favorite/Favorite.js]: Same here...

File: src/components/Favorite/Favorite.js [hunk 4]
```diff
@@ -21,27 +28,136 @@ export default function Favorite({
...
+                  <TbBuildingMonument />
+                </Style.TypeSpan>
+              )}
+              {type === 'sport' && (
+                <Style.TypeSpan>
+                  {' '}
```
Comment (JessicaLoers) [inline src/components/Favorite/Favorite.js]: here...

File: src/components/Favorite/Favorite.js [hunk 5]
```diff
@@ -21,27 +28,136 @@ export default function Favorite({
...
+              )}
+              {type === 'challenge' && (
+                <Style.TypeSpan>
+                  {' '}
+                  <GiMountainClimbing />
+                </Style.TypeSpan>
+              )}
+              {type === 'nature' && (
+                <Style.TypeSpan>
+                  {' '}
```
Comment (JessicaLoers) [inline src/components/Favorite/Favorite.js]: ... for the last time: here too.

File: src/components/Favorite/FavoriteStyle.js [hunk 6]
```diff
@@ -1,54 +1,104 @@
 import styled from 'styled-components';
 
-export const FavDiv = styled.div`
-  position: relative;
+export const Section = styled.section`
+  display: grid;
+  justify-items: start;
+  grid-template-columns: 2.3fr 1.5fr 0.2fr;
+  grid-template-rows: 1fr;
+  padding: 10px 10px;
```
Comment (JessicaLoers) [inline src/components/Favorite/FavoriteStyle.js]: Here you can use the short hand. `padding: 10px` But this is just a very oicky comment.

**Review by JessicaLoers — APPROVED** [review]
Really nice work and Design - it's 100/100 😁
Have just a few suggestions.

File: src/components/Favorite/Favorite.js (hunk 2 above)
Comment (bensle) [reply src/components/Favorite/Favorite.js]: Yep, vscode sets these lines. Thank you!

**Pull request merged**